### PR TITLE
Feature/tao 9050/thank you page

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/TAO-9050

- added a config entry for the return url: "return_url"
- added a complete.html file in each assets folder
- added a complete.js file, that manages the return link in complete.html file
- modified the bundle script to add the new javascript file

### CODE REVIEW

File: config/config.offline.sample.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "assignments": "http://localhost:8083/assignments",
     "lti": "http://localhost:8083/assignments/{assignmentId}/lti-link"
   },
+  "return_url": "/return-url",
```
Comment (jsconan) [inline config/config.offline.sample.json]: Should not it be `/index.html` by default?

Comment (bziondik) [reply config/config.offline.sample.json]: replaced

File: assets/tao/complete.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    </nav>
+                </header>
+        </div>
+        <div class="content-container delivery-complete">
+            <h1> Fine della valutazione </ h1>
+            <p> La valutazione è ora completa. </ p>
+            <p> Fare clic su "Termina il test" per confermare. </ p>
+            <p class = "congratulation"> Congratulazioni per la tua partecipazione. </ p>
+            <div class = "link-container">
+                <a class="end-test" href="#"> Termina il test </a>
```
Comment (jsconan) [inline assets/tao/complete.html]: For the default content, should not it be in english?

Comment (bziondik) [reply assets/tao/complete.html]: fixed

**Pull request merged**